rosie/core/__init__.py

```diff
@@ -28,7 +28,10 @@ def __init__(self, settings, adapter):
         self.settings = settings
         self.dataset = adapter.dataset
         self.data_path = adapter.path
-        self.suspicions = self.dataset[self.settings.UNIQUE_IDS].copy()
+        if self.settings.UNIQUE_IDS:
+            self.suspicions = self.dataset[self.settings.UNIQUE_IDS].copy()
+        else:
+            self.suspicions = self.dataset.copy()
```

Comment:
we will need to figure out a way to uniquely identify each federal senate reimbursements, otherwise the suspicions file will have all the columns that can be found in the original dataset... that's what we use the `UNIQUE_IDS` for, so if we are comfortable with all the columns in the suspicions file there's no reason to set an `UNIQUE_IDS`.